### DISCUSSION TITLE
Is Localisation allowed yet?

### DISCUSSION THREAD

**Discussion by SecularSteve** [grafana/grafana#53783] [General]
I'm proficient in Dutch, German and British English, and I was wondering where I need to go, and/or what files I need to open to translate this software to these languages. If it's even allowed.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
